apps/workspace-host/src/interface.js

```diff
@@ -628,6 +637,7 @@ async function _pullImage(workspace) {
         }
 
         const toDatabase = false;
+        cache.set(`workspaceProgressInit:${workspace_image}`, progressDetailsInit, 1000 * 60 * 60);
```

Comment:
`cache.set` is an async function; you should ensure errors are properly handled. See `workspaceUtils.updateWorkspaceMessage(...)` on the next line for how to do that.

Reply:
As Myles pointed out, `cache.set` is _not_ in fact an async function, it doesn't return anything at all!